static/app/views/dashboardsV2/datasetConfig/issues.tsx

```diff
@@ -24,6 +25,7 @@ const DEFAULT_WIDGET_QUERY: WidgetQuery = {
 export const IssuesConfig: DatasetConfig<never, Group[]> = {
   defaultWidgetQuery: DEFAULT_WIDGET_QUERY,
   getCustomFieldRenderer: getIssueFieldRenderer,
+  getTableFieldOptions: () => generateIssueWidgetFieldOptions(),
```

Comment:
```suggestion
  getTableFieldOptions: generateIssueWidgetFieldOptions,
```

Reply:
this is so ts doesn't complain about the inputs to `generateIssueWidgetFieldOptions`

Reply:
Oh! I didn't realize it had an optional parameter, my bad. Sounds good